src/components/Capture/Face.js

```diff
@@ -53,7 +53,7 @@ class Face extends Component {
   }
 
   renderUploadFallback = text =>
-    <CustomFileInput onChange={this.handleUpload} accept="image/*" capture="user">
+    <CustomFileInput className={style.faceCaptureFallback} onChange={this.handleUpload} accept="image/*" capture="user">
```

Comment:
Does the document live capture have the same bug? In that case we should fix it for that component too

Reply:
Good point. Will also update the acceptance criteria in the ticket so it gets checked also.